src/main/java/seedu/address/model/contact/Contact.java

```diff
@@ -107,6 +110,25 @@ public boolean isSameContact(Contact otherContact) {
                 && ((otherContact.getPhone().equals(getPhone()) || otherContact.getEmail().equals(getEmail())));
     }
 
+    /**
+     * Concatenates the contact data into a URL.
+     * @return URL of contact data.
+     * @throws MalformedURLException
+     */
+    public URL getUrlContactData() throws MalformedURLException {
```

Comment:
We just need the string, not the URL. See https://github.com/CS2103-AY1819S1-F10-3/main/blob/master/src/main/java/seedu/address/ui/BrowserPanel.java#L44-L46.